modules/post-installation-adding-nutanix-failure-domains-compute-machines.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/adding-nutanix-failure-domains.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="post-installation-adding-nutanix-failure-domains-compute-machines_{context}"]
+= Distributing compute machines across failure domains
+
+You can distribute compute machines across Nutanix failure domains by:
+
+* Modifying existing compute machine sets.
+* Creating new compute machine sets.
+
+The following procedure details how to distribute compute machines across failure domains by modifying existing compute machine sets. For more information on creating a compute machine set, see "Additional resources".
+
+.Prerequisites
+
+* You have configured the failure domains in the cluster's Infrastructure custom resource (CR).
```

Comment:
```suggestion
* You have configured the failure domains in the cluster's Infrastructure custom resource definition (CRD).
```

Reply:
This was raised early as part of an SME review. The decision was to go with CR.

https://github.com/openshift/openshift-docs/pull/69618#discussion_r1457598396